state/options/tx_options.go

```diff
@@ -0,0 +1,209 @@
+package options
```

Comment:
it was done to avoid circular dependency between blob and state, but I think i found a better solution: SubmitPFB will accept app.Blob instead of our custom(potentially it will be changed to go-square blob)

Reply:
moved options inside state package with some workaround